### DISCUSSION TITLE
blog/2024/05/03/birthday-spans-simulation-sans-math/

### DISCUSSION THREAD

**Discussion by giscus** [andrewheiss/ath-quarto#96] [Blog comments]
# blog/2024/05/03/birthday-spans-simulation-sans-math/

Probability math is hard. Use brute force simulation to find the probability that a household has a cluster of birthdays.

https://www.andrewheiss.com/blog/2024/05/03/birthday-spans-simulation-sans-math/

**Comment by brian-a-clark** [comment]
Have you considered accounting for non-singleton births (twins, triplets, etc)? 😉
